cypress/integration/models/applicationinventory/applicationinventory.ts

```diff
@@ -73,6 +88,94 @@ export class ApplicationInventory {
         });
     }
 
+    protected selectStakeholders(stakeholders: Array<string>): void {
+        stakeholders.forEach(function (stakeholder) {
+            selectFormItems(stakeholderSelect, stakeholder);
+        });
+    }
+
+    protected selectStakeholdergroups(stakeholdergroups: Array<string>): void {
+        stakeholdergroups.forEach(function (stakeholdergroup) {
+            selectFormItems(stakeholdergroupsSelect, stakeholdergroup);
+        });
+    }
+
+    protected selectMigrationAction(risk: string): void {
+        if (risk === "low") {
+            const migrationActions = ["Replatform", "Repurchase", "Retain"];
```

Comment:
Looks good so far @nitishSr , 

when risk is low I think we can Replatform , Refactor and Rehost and Retain.
When risk is high , options will be to Repurchase , Retire

Reply:
Made changes